internal/indexmanagement/scripts.go

```diff
@@ -419,11 +417,28 @@ for aliasBase in $writeAliases; do
 done
 `
 
+const deleteThenRolloverScript = `
+set -uo pipefail
+
+/tmp/scripts/delete
```

Comment:
these should be in the same directory as this script due to how we mount the configmap

Reply:
I am following the same path as we source the `/tmp/scripts/indexManagement` in these scripts, i.e. keep using absolute paths here